tests/_support/Helper/Acceptance/WooCommerce.php

```diff
@@ -59,11 +59,30 @@ public function setupWooCommercePlugin($I)
 				'payment_request_button_size'           => 'default',
 				'saved_cards'                           => 'no',
 				'logging'                               => 'no',
-				'upe_checkout_experience_enabled'       => 'disabled',
+				'upe_checkout_experience_enabled'       => 'yes',
 				'title_upe'                             => '',
 				'is_short_statement_descriptor_enabled' => 'no',
-				'upe_checkout_experience_accepted_payments' => [],
+				'upe_checkout_experience_accepted_payments' => [
```

Comment:
Update WooCommerce Stripe Gateway to use the 'new' checkout experience. Basically the same as before, with a different layout.